subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/InstrumentationPluginExt.kt

```diff
@@ -59,7 +59,8 @@ internal fun Project.withInstrumentationExtensionData(action: (InstrumentationPl
                         "reportViewerUrl" to extension.reportViewerUrl,
                         "sentryDsn" to extension.sentryDsn,
                         "slackToken" to extension.slackToken,
-                        "fileStorageUrl" to extension.fileStorageUrl
+                        "fileStorageUrl" to extension.fileStorageUrl,
+                        "teamcityBuildId" to envArgs.build.id.toString()
```

Comment:
Before that we will fail after test execution cause file for runner isn't generated